Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,448 @@
+import {Grid, GridItem, Box, Flex, Spacer, Text, Center, useColorMode, VStack, Image, SimpleGrid} from "@chakra-ui/react";
```

Comment:
```suggestion
import {Grid, GridItem, Box, Flex, Spacer, Text, Center, useColorMode, VStack, Image} from "@chakra-ui/react";
```
`SimpleGrid` is unused, so it can be removed